### DISCUSSION TITLE
RESTAPI /users/me returned only id

### DISCUSSION THREAD

**Discussion by gkibria** [directus/directus#15539] [Archive]
Hi, /users/me endpoint returned only id for user that has a role which has no app_access and admin_access.

for admin_access role or app_access role, endpoint returned data as expected.

How can I get all the users data for a role without app_access and admin_access?

Thanks

**Comment by rijkvanzanten** [comment]
Give the role access to read Directus Users. Add a permissions rule for `id` equals `$CURRENT_USER` to make sure that the user can only read their own data 👍🏻

**Reply by gkibria** [reply]
Thanks for the answer. it works!
